src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -1096,8 +1097,8 @@ export class ChatWidgetService implements IChatWidgetService {
 	private _widgets: ChatWidget[] = [];
 	private _lastFocusedWidget: ChatWidget | undefined = undefined;
 
-	get lastFocusedWidget(): ChatWidget | undefined {
-		return this._lastFocusedWidget;
+	get lastFocusedWidget(): IChatWidget | undefined {
```

Comment:
it's a bit weird that we call so many things `chatWidget` 🤔

Reply:
I think we should not call `TerminalChatController` `activeChatWidget`

Reply:
renamed things to be more specific.

Reply:
I also thought this reading it, but it makes sense. (I?)ChatWidget = the base used by all, (thing)ChatWidget is chat widget's embedded in the thing. Felt out of the scope of this so didn't mention and didn't have a good answer anyway.